tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -2154,6 +2162,9 @@ error: Cannot call an optional function
 error: Variable is not reassignable
    --> ../../../examples/tests/invalid/optionals.test.w:53:3
    |
+50 | if let hi = hi {
+   |        -- Variable is defined here (try adding \\"var\\" in front)
+   .
```

Comment:
Worth double checking if these errors look fine in the console.

Reply:
It was not automatically updated with the new annotations. Working on a fix